rojo.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "testez",
+  "servePort": 8000,
+  "partitions": {
+	"lib":{
+		"path":"lib",
+		"target":"ReplicatedStorage.TestEZ"
+	}
+  }
+}
```

Comment:
Your indentation is all messed up here. Probably better to just not check this file in this PR.